lib/main.dart

```diff
@@ -210,6 +188,34 @@ void _saveException(String? error, StackTrace? stackTrace) async {
   await file.writeAsString(jsonEncode(exception), mode: FileMode.append);
 }
 
+void _onError(FlutterErrorDetails details) {
+  print("#############");
+  print(details.exception.toString());
+  WidgetsBinding.instance.addPostFrameCallback(
+      (timeStamp) {
+        showPopUp<void>(
+          context: navigatorKey.currentContext!,
+          builder: (context) {
+            return AlertWithTwoActions(
+              isDividerExist: true,
+              alertTitle: S.of(context).error,
+              alertContent: "Oops, we got some error.\n\nPlease send crash report to our support team to make the application better.",
```

Comment:
Please send the crash report to our support team to make the application better.

(add `the`)